<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1245</width>
    <height>893</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="sliderBox">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>20</y>
      <width>380</width>
      <height>500</height>
     </rect>
    </property>
    <property name="title">
     <string>Manual Controls</string>
    </property>
    <widget class="FancySlider" name="X2_Slider">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>160</y>
       <width>260</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="FancySlider" name="Y1_Slider">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>100</y>
       <width>260</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="FancySlider" name="Z1_Slider">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>130</y>
       <width>260</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="FancySlider" name="X1_Slider">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>260</width>
       <height>20</height>
      </rect>
     </property>
     <property name="pageStep">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="FancySlider" name="Y2_Slider">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>190</y>
       <width>260</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="A1_Top_Label">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>45</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Field</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="FancySlider" name="Z2_Slider">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>220</y>
       <width>260</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="Y2_Label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>30</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Y-</string>
     </property>
    </widget>
    <widget class="QLabel" name="Z2_Label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>30</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Z-</string>
     </property>
    </widget>
    <widget class="QLabel" name="X1_Label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>30</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>X+</string>
     </property>
    </widget>
    <widget class="QLabel" name="X2_Label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>30</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>X-</string>
     </property>
    </widget>
    <widget class="QLabel" name="Y1_Label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>30</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Y+</string>
     </property>
    </widget>
    <widget class="QLabel" name="Z1_Label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>30</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Z+</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="Field_Current_Toggle">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>260</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Toggle Field/Current</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="X1_SpinBox">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>62</y>
       <width>50</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QSpinBox" name="Y1_SpinBox">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>93</y>
       <width>50</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QSpinBox" name="Z1_SpinBox">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>124</y>
       <width>50</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QSpinBox" name="Z2_SpinBox">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>217</y>
       <width>50</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QSpinBox" name="Y2_SpinBox">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>186</y>
       <width>50</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QSpinBox" name="X2_SpinBox">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>155</y>
       <width>50</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="csvInput1">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>430</y>
      <width>130</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>OpenFile</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1245</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCoil_Manipulator">
    <property name="title">
     <string>Coil Manipulator</string>
    </property>
   </widget>
   <addaction name="menuCoil_Manipulator"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>FancySlider</class>
   <extends>QSlider</extends>
   <header>fancyslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
